ros2/source/images.yaml.em

```diff
@@ -65,16 +84,15 @@ images:
             - flake8-quotes
             - pytest
             - pytest-cov
+            - pytest-repeat
+            - pytest-rerunfailures
             - pytest-runner
         ws: /root/ros2_ws
-        ament_args:
+        colcon_args:
             - build
-            - --build-tests
             - --cmake-args
-                -DSECURITY=ON --
-            - --isolated
-            - --parallel
+                " -DBUILD_TESTING=ON" " -DSECURITY=ON" " --no-warn-unused-cli"
```

Comment:
The leading spaces and quotes can be removed now.

Same below.

Reply:
Sweet :+1: